src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -205,11 +205,15 @@ export function getContainerFeaturesBaseDockerFile() {
 
 #{nonBuildKitFeatureContentFallback}
 
+FROM $_DEV_CONTAINERS_BASE_IMAGE AS dev_containers_feature_content_normalize
+COPY --from=dev_containers_feature_content_source {contentSourceRootPath} /tmp/build-features/
+RUN chmod -R 0600 /tmp/build-features
```

Comment:
Would this break features that come with additional executable files (beyond install.sh) because these won't be executable anymore? A simple fix could be to use `700`.

Another approach could be to copy the tar files to the build stage and untar there. That would preserve the original permissions.

Reply:
I think you're right. That's my bad; I didn't realize features were allowed to contain additional files. I've changed the `chmod` to `700`, since that's easiest and (I think?) harmless, but I can see how untaring inside docker would be more correct. If you want me to take a crack at that, let me know -- your call.

Reply:
We can go with this and look into copying the tars separately.